release-notes/opensearch-security-dashboards-plugin.release-notes-2.4.0.0.md

```diff
@@ -0,0 +1,26 @@
+## 2022-11-10 Version 2.4.0.0
+
+Compatible with Opensearch-Dashboards 2.4.0
+
+### Features
+* Initial commit for multiple authentication ([1110](https://github.com/opensearch-project/security-dashboards-plugin/pull/1110))
+* Saved Object Aggregation View ([1146](https://github.com/opensearch-project/security-dashboards-plugin/pull/1146))
+* [Saved Object Aggregation View] Use namespace registry to add tenant filter ([1169](https://github.com/opensearch-project/security-dashboards-plugin/pull/1169))
```

Comment:
Does this line has a leading space more than other rows? If so, please remove it to align with other rows.

Reply:
It only has one space at front.  The "[" makes it like two spaces.